internal/app/commands/sandbox/start.go

```diff
@@ -34,22 +34,6 @@ var Start = cli.Command{
 	Usage: "Start the sandbox.",
 	Action: func(c *cli.Context) error {
 
-		rData := common.ReadRuntimeData()
-		isSandboxRunning := common.VerifyRuntimeData(&rData)
-
-		port := uint16(c.Uint("http.port"))
-		if isSandboxRunning {
-			if rData.Running == c.Args().First() {
-				fmt.Fprintf(os.Stderr, "Sandbox '%s' is already running", rData.Running)
```

Comment:
Just out of curiosity: Is this check completely gone now or does the CLI check for whether the sandbox is already running somewhere else? I can't find the same check anywhere else in this change set.

Reply:
No, it's not gone :)
It exists in StartSandbox method if I recall correctly, which is invoked after all port checks, so I just removed duplication

Reply:
Ah, nice! 😄